metrics_listener.go

```diff
@@ -0,0 +1,57 @@
+package sundheitotel
+
+import (
+	"time"
+
+	gosundheit "github.com/AppsFlyer/go-sundheit"
+	"go.opentelemetry.io/otel/attribute"
+)
+
+type MetricsListener struct {
+	classification string
+	gauge          Int64Gauge
```

Comment:
I think I'd drop this, and replace with load/store of the results map and an observer